static/app/views/insights/settings.ts

```diff
@@ -169,3 +169,8 @@ export const MODULE_FEATURE_MAP: Record<ModuleName, string[]> = {
   [ModuleName.SCREEN_RENDERING]: SCREEN_RENDERING_MODULE_FEATURES,
   [ModuleName.OTHER]: [],
 };
+
+/**
+ * Modules that will not display tabs when the feature is not enabled
+ */
+export const MODULE_HIDDEN_WHEN_FEAUTRE_DISABLED: ModuleName[] = [];
```

Comment:
Hmm, i have an i an idea here, what if instead we add `MODULE_HIDDEN_FEATURE` here, for most modules this will be `module-entry-points`, but if you want the module to be hidden when the feature is disabled, we would just put the module-feature.

With this we could remove  `const showModuleTabs = organization.features.includes('insights-entry-points')` and rely purely on this object to hide module tabs. 

I like this because there isn't two places to look when trying to see if tab will be displayed or not.

Reply:
Ah yeah good call, let me adjust it to work like this